packages/react/src/Panels/EditorPanel.tsx

```diff
@@ -79,7 +79,7 @@ export function EditorPanel({
           </div>
         </div>
         <FileTree
-          className="flex-grow py-2 border-r border-tk-elements-app-borderColor text-sm"
+          className="flex-grow py-2 border-r border-tk-elements-app-borderColor text-sm overflow-y-auto overflow-x-hidden"
```

Comment:
This makes context menu's trigger area grow quite much now. This is visible in dev mode's `http://localhost:4321/1-basics/custom-chapter-slug/foo`:

![image](https://github.com/user-attachments/assets/5fdfef22-6ad3-48a8-b6f1-1267f81015df)

We'll need to add here

```suggestion
          className="flex flex-col flex-grow py-2 border-r border-tk-elements-app-borderColor text-sm overflow-y-auto overflow-x-hidden"
```